eng/common/scripts/stress-testing/java11-mvn-mariner-dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/openjdk/jdk:11-mariner as builder
+
+RUN yum -y install curl tar
```

Comment:
This should keep the image up to date on every build so we don't get OS vuln warnings later on:

```suggestion
RUN yum -y update
RUN yum -y install curl tar
```